source/_integrations/deako.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Deako Smart Lighting
+description: Instructions on how to integrate Deako Smart Lighting into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2024.10
+ha_domain: deako
+ha_config_flow: true
+ha_platforms:
+  - light
+ha_codeowners:
+  - '@sebirdman'
+  - '@balake'
+  - '@deakolights'
+ha_zeroconf: true
+ha_integration_type: integration
+---
+
+The **Deako Smart Lighting** integration allows you to control your [Deako](https://deako.com) devices from Home Assistant.
+
+## Prerequisites
+
+For this integration to work, your devices must be listed **online**, viewable in the Deako app. Once devices have been onboarded and are online, the Deako app is no longer needed. Devices will continue to work without the app.
+
+{% include integrations/config_flow.md %}
+
+## Supported devices
+
+- all smart Deako devices
+
+Features not currently supported:
+
+- Deako groups
+- Deako scenes
+
+This integration uses the [`pydeako`](https://pypi.org/project/pydeako/) library to connect to and interact with devices. One device is selected as the bridge over the local network to the Deako BLE mesh network, allowing for discovery of all devices and control.
```

Comment:
```suggestion
One device is selected as the bridge over the local network to the Deako BLE mesh network, allowing for discovery of all devices and control.
```
This note about the library used, is this needed? what would I do with this as an end user?

Reply:
It's not needed. I'll remove it.

Reply:
I need to remove the part about manual setup also as that's no longer applicable